common-npm-packages/build-scripts/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "build-scripts",
```

Comment:
should this file be updated ?

Reply:
I don't think so. I reverted the changes and built the task just fine